### ISSUE TITLE
Vector mask is ignored when a bitmap mask is present

### ISSUE THREAD

**Issue by sasensi** [Agamnentzar/ag-psd#54]
Hi thank you again for the awesome library !

In Photoshop, this is possible to have on a given layer, both a vector mask and a bitmap mask applied:
![Capture d’écran 2021-03-05 152836](https://user-images.githubusercontent.com/11247504/110128768-78961e00-7dc7-11eb-8534-3885a65b59da.png)

Taking this file as an example: [vector-and-bitmap-mask.zip](https://github.com/Agamnentzar/ag-psd/files/6091111/vector-and-bitmap-mask.zip)
![Capture d’écran 2021-03-05 152915](https://user-images.githubusercontent.com/11247504/110128863-8fd50b80-7dc7-11eb-90a1-5437f5e64380.png)
The circle with the play icon is made with a bitmap mask and the rectangle cropping is made with a vector mask.
If we try to achieve the same result using the library, only the vector mask is kept when opening the file in Photoshop.
Here is the code demonstrating my point:
```javascript
import fs from 'fs';
import { writePsdBuffer } from 'ag-psd';
import CanvasUtil from 'canvas';

// Switch value to compare both results.
const DISABLE_VECTOR_MASK = false;

function loadCanvasFromFile(filePath) {
  const img = new CanvasUtil.Image();
  img.src = fs.readFileSync(filePath);
  const canvas = CanvasUtil.createCanvas(img.width, img.height);
  canvas.getContext('2d').drawImage(img, 0, 0);
  return canvas;
}

const psd = {
  width: 101,
  height: 98,
  children: [
    {
      mask: {
        canvas: loadCanvasFromFile('mask.png'),
      },
      vectorFill: {
        type: 'color',
        color: {
          r: 255,
          g: 0,
          b: 0,
        },
      },
      ...(!DISABLE_VECTOR_MASK && {
        vectorMask: {
          paths: [
            {
              knots: [
                { points: [0, 0, 0, 0, 0, 0] },
                { points: [73, 0, 73, 0, 73, 0] },
                { points: [73, 74, 73, 74, 73, 74] },
                { points: [0, 74, 0, 74, 0, 74] },
              ],
            },
          ],
        },
      }),
    },
  ],
};

const buffer = writePsdBuffer(psd);
fs.writeFileSync('debug-write.psd', buffer);
```
And here is the bitmap mask image source referenced in the code as `'mask.png'`: 
![mask](https://user-images.githubusercontent.com/11247504/110129601-6b2d6380-7dc8-11eb-87fd-ef81ccc2ac4c.png)

Here is the produced file: [debug-write.zip](https://github.com/Agamnentzar/ag-psd/files/6091172/debug-write.zip)
![Capture d’écran 2021-03-05 153828](https://user-images.githubusercontent.com/11247504/110130024-dd05ad00-7dc8-11eb-8077-1a78e8ed55c7.png)
![Capture d’écran 2021-03-05 153835](https://user-images.githubusercontent.com/11247504/110130032-decf7080-7dc8-11eb-9e06-9e8c7b45a344.png)

Hoping that you'll be able to help with this.
Thank you in advance.

**Comment by Agamnentzar** [comment]
Fixed the issue as of version `11.6.0` 

Added `fromVectorData` field to mask object that indicates if the data is coming from vector data (in your case it will `false`)

**Comment by sasensi** [comment]
Thank you, I confirm that this perfectly fixes the issue !
